boards/cc2538dk/Makefile.include

```diff
@@ -3,7 +3,7 @@ export CPU        = cc2538
 export CPU_MODEL ?= cc2538nf53
 
 # setup serial terminal
-export PORT      ?= /dev/ttyUSB1
+PORT_LINUX ?= $(firstword $(wildcard /dev/serial/by-id/*XDS100v3*if01* /dev/ttyUSB1 /dev/ttyUSB*))
```

Comment:
The `/dev/ttyUSB*` seems pretty poinles to me as it will match either something connected prior to the cc2538dk or the first uart on the cc2538dk, which isn't the desired one. Do I miss something?